lightning-types/src/features.rs

```diff
@@ -59,13 +58,13 @@
 //! - `SCIDPrivacy` - supply channel aliases for routing
 //!   (see [BOLT-2](https://github.com/lightning/bolts/blob/master/02-peer-protocol.md) for more information).
 //! - `PaymentMetadata` - include additional data in invoices which is passed to recipients in the
-//!    onion.
-//!    (see [BOLT-11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) for
-//!    more).
+//!   onion.
+//!  (see [BOLT-11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) for
+//!  more).
```

Comment:
hmm these don't look right, though. does clippy complain if they have just one for leading space?